### DISCUSSION TITLE
SS2K suddenly spinning when bike not in ride mode

### DISCUSSION THREAD

**Discussion by StrangeMagic1** [doudar/SmartSpin2k#460] [General]
I have a pre-built SS2K. I use it with a peloton bike and I've had it for about six months. I typically would leave it on the bike because i only really ride with zwift.  I would try to unplug the adapter when not in use, but occasionally I would forget. Such happened recently. I noticed all of a sudden the SS2K started spinning by itself and making a grinding noise while my peloton was in sleep mode. I tried using it with the peloton and it didn't work.

I have reached out to the person I bought it from, but I was wondering if anybody has experienced this?

I tried rebooting, I restored to defaults.  Same problem.  I attempted to update the firmware by following the directions on this site, however, when I tried to upload the bin file from my computer, I was redirected to an error page so I don't think it worked.

Any help is appreciated. I haven't been using it that long… Maybe six months or so.

**Comment by eMadman** [comment]
I replied to your email separately, but... I just saw your comment re: firmware update failing.  Did you extract the firmware from the zip file or upload the downloaded zip?  

You can confirm the version that is installed on the smartspin2k.local home screen:
![image](https://user-images.githubusercontent.com/24726844/227530333-536c20ed-d822-4ecd-802e-36d62ee380ca.png)
